apps/frontend/src/libs/constants/sidebar-items.constant.ts

```diff
@@ -0,0 +1,15 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+import { SidebarItem } from "~/libs/types/types.js";
+
+const SIDEBAR_ITEMS: SidebarItem[] = [
+	{
+		href: AppRoute.ROOT,
+		icon: {
+			active: "myWheelActive",
+			inactive: "myWheelInactive",
+		},
+		title: "My Wheel",
+	},
+];
+
+export { sidebarItems };
```

Comment:
this should be `export { SIDEBAR_ITEMS }`

Reply:
It is exported like that on my local branch, I did a commit on GitHub itself on one of code suggestions, but it didn't go well